.azdo/pipelines/azure-dev.yml

```diff
@@ -67,6 +67,7 @@ steps:
       OPENAI_ORGANIZATION: $(OPENAI_ORGANIZATION)
       AZURE_USE_APPLICATION_INSIGHTS: $(AZURE_USE_APPLICATION_INSIGHTS)
       USE_VECTORS: $(USE_VECTORS)
+      USE_GPT4: $(USE_GPT4)
```

Comment:
Thanks for remembering the workflows! I need to update those in my embedding-3 PR.

Reply:
I just followed the `USE_GPT4v` changes 😅